### DISCUSSION TITLE
KafkaMirrormaker2: not all partitions get mirrored from a topic

### DISCUSSION THREAD

**Discussion by MandMaric** [strimzi/strimzi-kafka-operator#10877] [Q&A]
Hi everyone,

We are currently setting up our mm2. Right now we have the issues that not all partitions from a topic are getting mirrored. 3 out of 10 partitions are not getting mirrored at all. 

We do not see any resource (cpu, memory) related issues. Our assumption is that it might be related to taskMax.

Are there any best practices for mirrored partitions per task for the mirrormaker2?

**Comment by katheris** [comment]
Hi @MandMaric you can use a single task and it will still mirror all partitions. Do all the partitions contain data? You can see which tasks are assigned to which partitions using the status endpoint in the Connect REST API or take a look at the logs. The tasks should log their assignments at startup and when the assignment changes
